### PR TITLE
Support debugging dockerised Rawls in Intellij

### PR DESCRIPTION
**Pull request by ehigham**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check that the **Product Owner** has signed off on any user-facing changes
- [ ] **Submitter**: Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Database checks:
  * If PR includes new or changed db queries, include the explain plans in the description
  * Make sure liquibase is updated if appropriate
  * If doing a migration, take a backup of the
  [dev](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/backups?project=broad-dsde-dev&organizationId=548622027621)
  and
  [alpha](https://console.cloud.google.com/sql/instances/terraform-r4caezzc35c4tb7pgdhwkmme4y/backups?project=broad-dsde-alpha&organizationId=548622027621)
  DBs in Google Cloud Console
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] Tell your tech lead (TL) that the PR exists if they want to look at it
- [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via Slack and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -54,12 +54,11 @@ Global / excludeLintKeys += debugSettings // To avoid lint warning
...
 // itself rather than sbt, but they are not passed by default to the application, which is a forked
 // process. This passes them through to the "re-start" command, which is probably what a developer
 // would normally expect.
-sys.env.getOrElse("JAVA_OPTS", "").split(" ").toSeq.map { opt =>
-  javaOptions in reStart += opt
-}
+javaOptions in reStart ++= sys.env.getOrElse("JAVA_OPTS", "")
+  .concat(" -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5050")
+  .split(" ")
+  .toList
```
Comment (ehigham) [inline build.sbt]: If someone can tell me how to reorder the operations so I can remove the leading space before the `-agentlib` argument, I'll be very happy.

**Comment by ehigham** [comment]
@dmohs - I've requested a review from you as you had committed debug support to this file in the past. Feel free to pass the torch if you think someone else should review this.

File: build.sbt (hunk 1 above)
Comment (marctalbott) [reply build.sbt]:
```suggestion
javaOptions in reStart ++= sys.env.getOrElse("JAVA_OPTS", "")
  .split(" ")
  .toList :+ "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5050"
```

Have not confirmed that this works

**Review by marctalbott — APPROVED** [review]

**Review by dmohs — APPROVED** [review]
I appreciate the heads up and I'm happy to add both my approval and appreciation for tooling improvements, for what those are worth.

**Pull request merged**